bundles/org.openhab.binding.nanoleaf/src/main/resources/OH-INF/i18n/nanoleaf_de.properties

```diff
@@ -17,8 +17,8 @@ thing-type.config.nanoleaf.controller.authToken.description = Token zur Authenti
 thing-type.config.nanoleaf.controller.refreshInterval.label = Aktualisierungsintervall
 thing-type.config.nanoleaf.controller.refreshInterval.description = Intervall (in Sekunden) in dem die Kanäle aktualisiert werden
 thing-type.config.nanoleaf.controller.deviceType.label = Nanoleaf Gerätetyp
-thing-type.config.nanoleaf.controller.deviceType.description = Light Panels (Dreiecke) oder Canvas (Quadrate)
-thing-type.config.nanoleaf.lightpanel.id.label = Paneel ID
+thing-type.config.nanoleaf.controller.deviceType.description = Light Panels (alte Dreieck-Modelle) oder Canvas/Shapes (neue Modelle als Quadrat, Dreieck, Hexagon, etc.)
+thing-type.config.nanoleaf.lightpanel.id.label = Panel ID
```

Comment:
Can you please change the German translations later in Crowdin ?
If you do it here, someone will have to take care to manually load your changes in Crowdin. If not done, this could lead later to your translations being overwritten.

Or I can merge the PR if you load your file in Crowdin just after ?

Reply:
I have also done the changes in Crowdin (they just need to be approved there).

Reply:
Ok, let's go for a merge.
@cweitkamp to be sure that there will be no mismatch with German translation.

Reply:
I approved the changes in Crowdin. Thanks for fixing.